django/contrib/admin/templates/admin/includes/fieldset.html

```diff
@@ -22,7 +22,7 @@
                 </div>
                 {% if field.field.help_text %}
                     <div class="help"{% if field.field.id_for_label %} id="{{ field.field.id_for_label }}_helptext"{% endif %}>
-                        {{ field.field.help_text|safe }}
+                        <p>{{ field.field.help_text|safe }}</p>
```

Comment:
Adding `<p>` changes font size and few other things, we probably cannot use it here :disappointed:
Django 4.1:
![image](https://user-images.githubusercontent.com/2865885/218392952-9c6e1a45-3821-4b14-914b-697d83774f02.png)

This patch:
![image](https://user-images.githubusercontent.com/2865885/218392831-bc40961c-728a-4ba5-b362-e3a465ac4205.png)